docs/source/tutorials/token_classification/utils.py

```diff
@@ -0,0 +1,242 @@
+import string 
```

Comment:
Move these functions as we discussed: 

1) If intended to be part of library and user-facing, put in:
cleanlab/token_classification/summary.py

2) If intended to be part of library and not user-facing (helper methods used by other user-facing methods of the library), put in:
cleanlab/internal/token_classification_utils.py

3) If only useful for the entity_recognition_training.ipynb notebook, put in:  docs/source/tutorials/token_classification/run_ner.py (try to minimize the number of scripts in this folder and put as much of the pure python code in a single file if you can)

Reply:
Option 1 includes any functions directly called in token_classification.ipynb tutorial.  All of these functions must be very clearly documented since they are user-facing, with proper type annotations for their arguments and what they return.

Reply:
Option 2 includes functions that are called inside of any of the existing cleanlab.token_classification.rank functions as well as inside any other functions classified as part of Option 1.

Reply:
Option 3 includes functions that are only called in the model-training notebook and nowhere else.

Reply:
All functions should have some documentation. The functions in Option 1 should have very detailed and polished documentation. The functions in Option 2 should have somewhat detailed documentation. The functions in Option 3 can just have simple documentation (eg. 1-2 sentences) if you are in a time-crunch.